lib/addmagma.gi

```diff
@@ -68,33 +68,37 @@ InstallMethod( ViewObj,
     "for an add. magma with generators",
     [ IsAdditiveMagma and HasGeneratorsOfAdditiveMagma ],
     function( A )
+    local nrgens;
+    nrgens := Length( GeneratorsOfAdditiveMagma( A ) );
     Print( "<additive magma with ",
-           Length( GeneratorsOfAdditiveMagma( A ) ), " generators>" );
+           nrgens, " ", Pluralize( nrgens, "generator" ), ">" );
```

Comment:
I wonder: are there any places where `Pluralize` is not used like that? Perhaps it'd make more sense then to have `Pluralize` instead return this, i.e. a string like `"123 generators"` ? 

And if somebody *really* just wants the plural, well, they don't need a specific count, right? So perhaps the first argument `count` could be optional, too, and if omitted it just returns the plural. 

I just had a look at how Ruby on Rails does it, and it turns out it's exactly like what I just suggested :-)